detox/src/devices/drivers/AndroidDriver.js

```diff
@@ -252,6 +252,12 @@ class AndroidDriver extends DeviceDriverBase {
       return result;
     }, []);
   }
+
+  async setStatusBar(deviceId, flags) {
```

Comment:
According to OOP common sense, you should have defined them in the base driver class.

Reply:
Fixed